scripts/metamath-extract.py

```diff
@@ -214,7 +217,7 @@ def abbreviate(name: str) -> str:
 
 def abbreviate_constructors(term: Term) -> Term:
     if isinstance(term, Metavariable):
-        return term
+        return Metavariable(abbreviate(term.name))
```

Comment:
I did put it in a standalone commit, though I wish we could merge the PR without squashing commits. This PR needs this fix because, the new changes  do not verify without it. 

https://github.com/fiedlr/k-proof-generation/pull/17/commits/1453bcbfb3f0619ea1f25d7048d6b0e6e7f9a722